FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -127,26 +127,26 @@ fi
 opts_SetScriptDescription "Runs the FreeSurfer HCP pipline on data processed by prefresurfer"
 
 # Show usage information
-opts_AddMandatory '--subject' 'SubjectID' 'subject' "Subject ID (required).  Used with --path input to create full path to root directory for all outputs generated as path/subject"
+opts_AddMandatory '--session' 'SessionID' 'session' "Session ID (required).  Used with --path input to create full path to root directory for all outputs generated as path/session"
```

Comment:
This is missing the compatibility flag.

```suggestion
opts_AddMandatory '--session' 'SessionID' 'session' "Session ID (required).  Used with --path input to create full path to root directory for all outputs generated as path/session" '--subject'
```

Reply:
fixed